libs/core/src/lib/token/tokenizer.component.ts

```diff
@@ -169,14 +170,15 @@ export class TokenizerComponent implements AfterViewInit, AfterContentInit {
     /** @hidden */
     handleKeyDown(event: KeyboardEvent, fromIndex: number): void {
         let newIndex: number;
-        if (event.code === 'ArrowLeft') {
+        const rtl = this._rtlService.rtl.getValue();
```

Comment:
Won't you get `property of undefined` error if there will be no `RtlService` provided?